keps/sig-scheduling/5027-dra-admin-controlled-device-attributes/README.md

```diff
@@ -302,22 +290,30 @@ type DevicePatch struct {
     // +optional
     // +featureGate:DRAAdminControlledDeviceAttributes
     Capacity map[FullyQualifiedName]DeviceCapacity
+
+    // ^^^^
+    // The assumption here is that all device types will have attributes and capacities,
+    // similar to the current BasicDevice type. Therefore the overrides are not made
+    // specific to certain device types.
 }
 
 // DevicePatchFilter defines which device(s) a [DevicePatch] applies to.
 type DevicePatchFilter struct {
```

Comment:
It may be something out of the scope of this KEP, can an external controller add a selector that can be later consumed by the machinery described in the KEP?

Reply:
This touches on the question whether `DevicePatchFilter` is immutable: it isn't, so whenever a selector gets added or removed, it changes how the scheduler evaluates the patch.

The machinery in this KEP doesn't care who does the updating, so it could be a controller.